tools/wptrunner/wptrunner/formatters.py

```diff
@@ -1,8 +1,55 @@
 import json
+import re
+import sys
 
 from mozlog.structured.formatters.base import BaseFormatter
 
 
+LONE_SURROGATE_RE = re.compile(u"[\uD800-\uDFFFF]")
+
+
+def surrogate_replacement_ucs4(match):
+    return "U+" + hex(ord(match.group()))[2:]
```

Comment:
This always outputs six characters, right? Given the match is always a single surrogate code point?